<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-04-19T11:50:32 IST" name="Default suite" finished-at="2022-04-19T11:50:40 IST" duration-ms="8253">
    <groups>
    </groups>
    <test started-at="2022-04-19T11:50:32 IST" name="Default test" finished-at="2022-04-19T11:50:40 IST" duration-ms="8253">
      <class name="RestAssured_BasicApi.TC003_PUT_UpdateUserData">
        <test-method signature="getUserData()[pri:0, instance:RestAssured_BasicApi.TC003_PUT_UpdateUserData@ea6147e]" started-at="2022-04-19T11:50:32 IST" name="getUserData" finished-at="2022-04-19T11:50:32 IST" duration-ms="46" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[Cannot invoke "io.restassured.response.Response.getBody()" because "this.response" is null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: Cannot invoke "io.restassured.response.Response.getBody()" because "this.response" is null
at RestAssured_BasicApi.TC003_PUT_UpdateUserData.getUserData(TC003_PUT_UpdateUserData.java:39)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:808)
at org.testng.TestRunner.run(TestRunner.java:603)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getUserData -->
        <test-method signature="userAdd()[pri:1, instance:RestAssured_BasicApi.TC003_PUT_UpdateUserData@ea6147e]" started-at="2022-04-19T11:50:32 IST" name="userAdd" finished-at="2022-04-19T11:50:40 IST" duration-ms="8123" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- userAdd -->
      </class> <!-- RestAssured_BasicApi.TC003_PUT_UpdateUserData -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
